djautotask/api_rest.py

```diff
@@ -514,11 +513,11 @@ class AccountTypesAPIClient(AutotaskPicklistAPIClient):
     API_ENTITY = 'Companies'
 
 
-class TicketsCategoryAPIClient(AutotaskPicklistAPIClient):
+class TicketCategoryPicklistAPIClient(AutotaskPicklistAPIClient):
     API_ENTITY = 'TicketCategories'
 
 
-class TicketsPicklistAPIClient(AutotaskPicklistAPIClient):
+class TicketPicklistAPIClient(AutotaskPicklistAPIClient):
```

Comment:
I'm a little curious about these two renaming changes you did here. Why did you change them? They don't follow the same pattern as the rest of the picklist API clients now.